components/time-picker/demo/need-confirm.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import type { TimePickerProps } from 'antd';
+import { TimePicker, Space } from 'antd';
+
+const onChange: TimePickerProps['onChange'] = (time, timeString) => {
+  console.log(time, timeString);
+};
+
+const App: React.FC = () => (
+  <Space direction="vertical" size={12}>
+    <TimePicker onChange={onChange} needConfirm />
+    <TimePicker onChange={onChange} needConfirm={false} />
+  </Space>
```

Comment:
```suggestion
  <TimePicker onChange={onChange} needConfirm />
```

Reply:
- First of all, thank you for suggesting the code :)
- There were two reasons I included an example specifying both options.
- First, it was to maintain consistency with the example format of the datePicker's needConfirm, which provides similar functionality. ([Related PR](https://github.com/ant-design/ant-design/pull/47275/files#diff-7ad15fc669c0848d87e71f9151c75d5af435c4d256181fbebe839e892469e847))
 
<img width="1306" alt="스크린샷 2024-05-17 오후 3 51 28" src="https://github.com/ant-design/ant-design/assets/37934668/bce14b22-c87a-44be-8444-1e2f5cd79b18">

<br/>

- Second, since the demos of the timePicker are set with needConfirm as true, I wanted to show the false option.

<img width="1292" alt="스크린샷 2024-05-17 오후 3 48 54" src="https://github.com/ant-design/ant-design/assets/37934668/1b6dbfab-9e7b-4c87-9cea-636e1c29b593">

<br/><br/>

- I structured the example for these two reasons, and I would appreciate your feedback on this :)

Reply:
It is hard to tell these two timepickers from appearance.

And `needConfirm={false}` is the default behaviour which people can compare with other demos.

Reply:
I agree that the operation can be described with just `needConfirm={false}`. (updated: https://github.com/ant-design/ant-design/pull/48945/commits/90acb853daa4a7420947e13211b3f1facf8198aa)
However, I would like to ensure uniformity in the demo. 😢  
Would it be okay to request a separate PR to modify the datePicker demo to match the format of the timePicker demo?
If you okay, i request PR about changed datePicker demo.

Reply:
Sure~

Reply:
I request date-picker update docs PR 🕺 https://github.com/ant-design/ant-design/pull/48967